src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -2261,7 +2315,98 @@ public static void copyImplementation(Design src, Design dest, boolean lockPlace
 					p.setIsPIPFixed(true);
 				}
 			}
-		}		
+		}
+
+		if (copyStaticNet) {
+			copyStaticNets(dest, staticNets, siteInstsOfCells);
+		}
+	}
+
+	private static boolean equalNodes(Node lNode, Node rNode) {
+		return (lNode.getTile().getName().equals(rNode.getTile().getName()) && lNode.getWireName().equals(rNode.getWireName()));
+	}
```

Comment:
Why not use `Node.equals()`?  This seems like it would be slower/redundant.

Reply:
Node.equals() uses wire != other.wire.  Is that  object address comparison?

At least I can use Tile.equals().

Reply:
Changed to Node.equals().